### PR TITLE
Gallery fixes for 1.0

### PR DESCRIPTION
**Pull request by ehsteve**
### Description

Fixes #<Issue Number>

### CODE REVIEW

File: examples/map/image_bright_regions_gallery_example.py [hunk 1]
```diff
@@ -1,56 +1,36 @@
...
 import sunpy.map
 from sunpy.data.sample import AIA_193_IMAGE
 
-##############################################################################
-# Now, we create a SunPy Map object from an AIA FITS file.
-
+###############################################################################
+# We start with the sample data
 aiamap = sunpy.map.Map(AIA_193_IMAGE)
+aiamap_orig = sunpy.map.Map(AIA_193_IMAGE)
```
Comment (Cadair) [inline examples/map/image_bright_regions_gallery_example.py]: errrr?

Comment (Cadair) [reply examples/map/image_bright_regions_gallery_example.py]: We should never modify a map in place, so we really shouldn't have to do this.

File: examples/map/image_bright_regions_gallery_example.py [hunk 2]
```diff
@@ -60,38 +40,33 @@
...
 
 ##############################################################################
 # Finally, we plot the smoothed bright image data, along with the estimate of
 # the number of distinct regions. We can see that approximately 6 distinct hot
 # regions are present above the 5% of the maximum level.
-
 plt.figure()
-aiamap2.plot()
+ax = plt.subplot(projection=aiamap)
+aiamap_orig.plot()
```
Comment (Cadair) [inline examples/map/image_bright_regions_gallery_example.py]: this should just be `aiamap`

File: examples/map/map_resampling_and_superpixels.py [hunk 3]
```diff
@@ -1,63 +1,47 @@
...
+aia_resampled_map = aia_map.resample(new_dimensions)
 
-fig = plt.figure()
+##############################################################################
+# Let's plot the result.
+ax = plt.subplot(projection=aia_resampled_map)
 aia_resampled_map.plot()
-aia_resampled_map.draw_grid()
-aia_resampled_map.draw_limb()
+plt.show()
```
Comment (Cadair) [inline examples/map/map_resampling_and_superpixels.py]: show here?

File: examples/units_and_coordinates/SDO_to_STEREO_Coordinate_Conversion.py [hunk 4]
```diff
@@ -127,13 +118,14 @@
 # We can now zoom in on the region in the EUVI image:
 subeuvi = maps['EUVI'].submap(hpc_B[0], hpc_B[3])
 fig = plt.figure()
+fig.add_subplot(111, projection=subeuvi)
```
Comment (Cadair) [inline examples/units_and_coordinates/SDO_to_STEREO_Coordinate_Conversion.py]:
```suggestion
plt.subplot(projection=subeuvi)
```

File: examples/acquiring_data/2011_06_07_sampledata_overview.py [hunk 5]
```diff
@@ -31,13 +31,16 @@
 # The following image of the flare is now fairly iconic.
 aia_cutout03_map = sunpy.map.Map(sample_data.AIA_193_CUTOUT03_IMAGE)
 fig = plt.figure()
+ax = fig.add_subplot(111, projection=aia_cutout03_map)
```
Comment (Cadair) [inline examples/acquiring_data/2011_06_07_sampledata_overview.py]:
```suggestion
ax = plt.subplot(projection=aia_cutout03_map)
```

File: examples/map/image_bright_regions_gallery_example.py (hunk 1 above)
Comment (ehsteve) [reply examples/map/image_bright_regions_gallery_example.py]: but we do in this example...I didn't even think we could!

File: examples/computer_vision_techniques/loop_edge_enhance.py [hunk 6]
```diff
@@ -16,30 +17,27 @@
...
 edge_map = sunpy.map.Map(edge_enhanced_im, aia_smap.meta)
 
-fig = plt.figure(figsize=(12, 6))
-ax = fig.add_subplot(121, projection=aia_smap)
-aia_smap.plot()
-ax = fig.add_subplot(122, projection=aia_smap)
+###############################################################################
+# Let's plot the results.
+fig = plt.figure()
+ax = fig.add_subplot(111, projection=aia_smap)
```
Comment (Cadair) [inline examples/computer_vision_techniques/loop_edge_enhance.py]:
```suggestion
ax = plt.subplot(projection=aia_smap)
```

File: examples/map/image_bright_regions_gallery_example.py (hunk 2 above)
Comment (ehsteve) [reply examples/map/image_bright_regions_gallery_example.py]: I changed it on purpose. Showing the original map with the contours is more illustrative than showing the blurred out map.

File: examples/map/map_resampling_and_superpixels.py (hunk 3 above)
Comment (ehsteve) [reply examples/map/map_resampling_and_superpixels.py]: ?

File: examples/computer_vision_techniques/off_limb_enhance.py [hunk 7]
```diff
@@ -70,28 +63,24 @@
...
 
-fig = plt.figure(figsize=(12, 5))
-ax = fig.add_subplot(121, projection=aia)
-aia.plot()
-aia.draw_limb()
-ax = fig.add_subplot(122, projection=aia)
+###############################################################################
+# Let's plot the results
+fig = plt.figure()
+ax = fig.add_subplot(111, projection=aia)
```
Comment (Cadair) [inline examples/computer_vision_techniques/off_limb_enhance.py]:
```suggestion
ax = plt.subplot(projection=aia)
```

File: examples/map/submaps_and_cropping.py [hunk 8]
```diff
@@ -1,50 +1,34 @@
...
 # left as SkyCoord objects.
-
 top_right = SkyCoord(0 * u.arcsec, -200 * u.arcsec, frame=swap_map.coordinate_frame)
 bottom_left = SkyCoord(-900 * u.arcsec, -900 * u.arcsec, frame=swap_map.coordinate_frame)
 swap_submap = swap_map.submap(bottom_left, top_right)
 
+###############################################################################
+# Let's plot the results
 fig = plt.figure()
+ax = fig.add_subplot(111, projection=swap_submap)
```
Comment (Cadair) [inline examples/map/submaps_and_cropping.py]:
```suggestion
ax = plt.subplot(projection=swap_submap)
```

File: examples/plotting/wcs_axis_map_example.py [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+###############################################################################
+# By setting the projection of the axis, a WCSAxes is created and this enables
+# the plot to be created with the map coordinates, in this case helioprojective
+# coordinates. The standard plot
+# command expects pixel coordinates. It is also possible to plot a point using the
+# map coordinates if we pass the transformation but this transform expects
+# the values to be in degrees and not arcseconds.
+fig = plt.figure()
+ax = fig.add_subplot(1, 1, 1, projection=smap)
```
Comment (Cadair) [inline examples/plotting/wcs_axis_map_example.py]:
```suggestion
ax = plt.subplot(projection=smap)
```

File: examples/units_and_coordinates/SDO_to_STEREO_Coordinate_Conversion.py [hunk 10]
```diff
@@ -127,13 +117,13 @@
 # We can now zoom in on the region in the EUVI image:
 subeuvi = maps['EUVI'].submap(hpc_B[0], hpc_B[3])
 fig = plt.figure()
+plt.subplot(projection=subeuvi)
 subeuvi.plot()
 
 ###############################################################################
 # Putting them together:
-fig = plt.figure(figsize=(15, 5))
+fig = plt.figure()
```
Comment (Cadair) [inline examples/units_and_coordinates/SDO_to_STEREO_Coordinate_Conversion.py]: no don't take my figsize away!!
```suggestion
fig = plt.figure(figsize=(15, 5))
```

File: examples/units_and_coordinates/SDO_to_STEREO_Coordinate_Conversion.py [hunk 11]
```diff
@@ -111,10 +101,10 @@
 
 ###############################################################################
 # Now we can plot this box on both the AIA and EUVI images:
-fig = plt.figure(figsize=(10, 4))
+fig = plt.figure()
```
Comment (Cadair) [inline examples/units_and_coordinates/SDO_to_STEREO_Coordinate_Conversion.py]:
```suggestion
fig = plt.figure(figsize=(10, 4))
```

File: examples/plotting/hmi_synoptic_maps.py [hunk 12]
```diff
@@ -20,44 +17,33 @@
...
-# Set the colorbar properties.
+# Let's also fix the plot settings
 syn_map.plot_settings['cmap'] = 'hmimag'
 syn_map.plot_settings['norm'] = plt.Normalize(-1500, 1500)
 
 ###############################################################################
-# Create a figure with the Map's projection:
-fig = plt.figure(figsize=(12, 5))
+# Let's plot the results
+fig = plt.figure()
```
Comment (Cadair) [inline examples/plotting/hmi_synoptic_maps.py]:
```suggestion
fig = plt.figure(figsize=(12, 5))
```

File: examples/acquiring_data/searching_vso.py [hunk 13]
```diff
@@ -1,45 +1,40 @@
...
 print(downloaded_files)
 
 ###############################################################################
 # More complicated queries can be constructed by using relational operators.
-# For example, say we are interested in both eit and mdi data.
-
-result = Fido.search(a.Time('2012/3/4', '2012/3/6'),
+# For example, it is possible to query two wavelengths at the same time with
+# the OR operator (|).
+result = Fido.search(a.Time('2012/03/04 00:00', '2012/03/04 00:02'),
```
Comment (Cadair) [inline examples/acquiring_data/searching_vso.py]: two seconds? That's not very long?

Comment (ehsteve) [reply examples/acquiring_data/searching_vso.py]: 2 minutes. that already returns 25 images!

**Pull request merged**